firebase-messaging/src/main/java/com/google/firebase/messaging/GmsRpc.java

```diff
@@ -58,6 +58,11 @@ class GmsRpc {
   /** Another server error besides ERROR_SERVICE_NOT_AVAILABLE that we retry on. */
   static final String ERROR_INTERNAL_SERVER_ERROR = "INTERNAL_SERVER_ERROR";
 
+  /** A server error that represents hitting topic subscription quota. Trying again here may
```

Comment:
This should start on a separate line (like the other multi-line Javadoc in this file).

Reply:
Done